tests/pcap-file-delete-non-alerts-only-no-alerts/suricata.yaml

```diff
@@ -0,0 +1,7 @@
+%YAML 1.1
+---
+
+include: ../alert-testmyids/default.yaml
+
+pcap-file:
```

Comment:
test.yaml can also have an args : `--set pcap-file.delete-non-alerts-only = true` (or something like this) if you prefer